packages/composables/src/useCart/useCart.ts

```diff
@@ -66,8 +66,14 @@ export type UseCartReturn = {
   totalPrice: ComputedRef<number>;
   /**
    * Shipping price
+   *
+   * @deprecated Use `shippingTotal` instead
```

Comment:
looks like this is in the wrong line? You say to use the thing which becomes deprecated ;)

Reply:
ups :D fixed